### ISSUE TITLE
Processes crashing when PNC cutoff does not converge 

### ISSUE THREAD

**Issue by landmanbester** [landmanbester/Copernicus#8]
This is driving me crazy. Sometimes (i.e. when we have shitty data) the PNC cutoff calculation does not converge. I am raising a flag and exiting Fortran when this happens. When I get back to python though the process just hangs without warning or apparent reason. As far as I can tell this happens when we have large values of Lambda and small values of rho at the vertex.

@julienlarena can you think of a constraint other than Om + OL + OK = 1 that we can check at the vertex?

**Comment by julienlarena** [comment]
We could use an estimate of \Omega_{m} in the local Universe, maybe?
But that would depend on H0, so I'm not sure it is a good idea. On the
other hand, there has to be a constraint on \rho local?

**Comment by landmanbester** [comment]
Yeah there should be but not one that doesn't depend on a lot of astrophysical assumptions. I have managed to flag these samples and will just discard them for now. It doesn't happen when I use redshift drift data, only current

**Comment by julienlarena** [comment]
Ok.
